File: ios/LaneShadow/Sandbox/Stories/MoleculesStories.swift
import LaneShadowTheme
import NativeSandbox
import SwiftUI

/// Molecule stories — Sprint 4 molecule components.
@MainActor
enum MoleculesStories {
    static let all: [Story] = {
        var stories: [Story] = []
        // stories += LSContentCardStory.all
        // stories += LSListRowStory.all
        stories += LSToolbarStory.all
        // stories += LSNavHeaderStory.all
        stories += LSBottomSheetStory.all
        stories += LSToastStory.all
        stories += LSModalStory.all
        stories += LSFormFieldStories.all
        stories += LSTabItemStories.all
        stories += LSEmptyStateStories.all
        // stories += LSPillSemanticsStory.all
        // stories += LSLocationContextBarStory.all
        // stories += LSRouteAttachmentCardStory.all
        stories += LSChatInputStories.all
        // stories += LSNavigatorMoleculesStories.all
        return stories
    }()
}

File: android/app/src/debug/java/com/laneshadow/sandbox/stories/molecules/MoleculesStories.kt
package com.laneshadow.sandbox.stories.molecules

import com.nativesandbox.model.Story

object MoleculesStories {
    val all: List<Story> =
        LSContentCardStory.all +
            LSListRowStory.all +
            LSToolbarStory.all +
            LSNavHeaderStory.all +
            LSBottomSheetStory.all +
            LSToastStory.all +
            LSModalStory.all +
            LSFormFieldStory.all +
            LSAuthProviderButtonStory.all +
            LSTabItemStory.all +
            LSEmptyStateStory.all +
            LSPillSemanticsStory.all +
            LSLocationContextBarStory.all +
            LSRouteAttachmentCardStory.all +
            LSChatInputStory.all +
            LSContextCapsuleStory.all +
            LSIdleHeaderStory.all +
            LSNavigatorMoleculesStory.all
}
